source/stylesheets/refills/_badges.scss

```diff
@@ -1,36 +1,20 @@
-$badge-font-size: 0.75em !default;
-$medium-gray: #999 !default;
-$alert-color: #fff6bf !default;
-$error-color: #fbe3e4 !default;
-$notice-color: #e5edf8 !default;
-$success-color: #e6efc2 !default;
+$small-font-size: 0.75em;
+$badges: (
+  default: #999,
+  alert: #fff6bf,
+  error: #fbe3e4,
+  notice: #e5edf8,
```

Comment:
Color literals like `#e5edf8` should only be used in variable declarations; they should be referred to via variable everywhere else.